main_test.go

```diff
@@ -27,9 +27,7 @@ func TestMain(m *testing.M) {
 
 func setupIntegrationTests() {
 	os.Setenv(cliutils.ReportUsage, "false")
-	os.Setenv(cliutils.OfferConfig, "false")
-	// Disable progress bar:
```

Comment:
Let's keep the comment and add :
// Disable progress bar and confirmation messages